### ISSUE TITLE
Icon Button - Need to add light variant to properties

### ISSUE THREAD

**Issue by ArakTaiRoth** [GovAlta/ui-components#1848]
## Acceptance Criteria

1. Need to add `light` type to `variant` property
2. Need to update Sandbox to allow selection of `light` variant
3. Remove `nocolor` type from `variant` property
4. Update Sandbox to remove `nocolor` variant

## More Info

This can only be done after #1146 is pushed to Production

**Comment by ArakTaiRoth** [comment]
Closing as this was already done by @bdfranck and I just didn't notice it until now
